database/postgres/postgres.go

```diff
@@ -187,10 +200,17 @@ func (p *Postgres) Run(migration io.Reader) error {
 	if err != nil {
 		return err
 	}
-
+	var ctx context.Context
```

Comment:
simplify to:
```golang
ctx := context.Background()
if p.config.StatementTimeout == 0 {
    var cancel context.CancelFunc
    ctx, cancel = context.WithTimeout(ctx, p.config.StatementTimeout)
    defer cancel()
}
```

Reply:
sg tiny change `if p.config.StatementTimeout != 0` instead

Reply:
done